README.md

```diff
@@ -53,6 +53,22 @@ generate a key, with optional seed.
 curve defaults to `ed25519` (and no other type is currently supported)
 seed should be a 32 byte buffer.
 
+Output is in the form
+
+    {
+      "curve": "ed25519", // Our current eliptic curve
+      "public": "fHlqm8WWP4ODtsuXazhJR66YrmhOWcmqae4LxXPHn0o=.ed25519", // format: public_key.curve
```

Comment:
I'm going to merge this but change 

`public_key.curve` >  `key.curve` because the former makes it confusing about whether or not the curve is part of the public key. mostly if people say "give me your public key" they mean `key.curve`